db/migrate/20200214150738_add_password_digest_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddPasswordDigestToUsers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :password_digest, :string
```

Comment:
Can this migration be merged with the migration for create users table?

Reply:
Uhm I'm not sure about this